### ISSUE TITLE
Mobile gap between the draggable element and finger on a scaled container

### ISSUE THREAD

**Issue by ndenkha** [RWAP/jquery-ui-touch-punch#34]
Hi,

I downloaded this version hoping it would fix this issue in 0.2.3 but no luck.
I can't find any fix to this, yet all blogs point to jQuery touch punch.

Are you aware of this issue? where when on a mobile device, dragging an element onto a scaled-down container causes a gap between the dragged element and the finger?
The scale rate I have is 0.45, but the behavior exists as long as the value is under 1.

Thanks,

Ninos

**Comment by RWAP** [comment]
I am not sure how jQuery Touch Punch would actually cause this - the code is designed to interpret various finger movements / actions on a touch screen device and convert them to a mouse movement / action.  How a container is then dragged and inserted is pretty much down to jQuery itself.

If you had some working code to test, someone might be able to spot what is causing the issue.  Does it do the same on a desktop using mouse (compared to touch screens)?

**Comment by ndenkha** [comment]
No, the desktop works perfectly. This only started happening after we scaled down the container for mobile view.
Unfortunately, I don't have a working sample that produces the issue, if I get time I can work on mocking something and sharing it with you.
Thank you for getting back to me.
